requirements.txt

```diff
@@ -83,6 +83,7 @@ snowflake-connector-python==3.2.1
 sshtunnel==0.4.0
 tables==3.7.0
 trino==0.321.0
+weaviate-client>=3.25.3
```

Comment:
let's lock the `weaviate-client` version

Reply:
Good point. Locked it